internal/broadcaster/rate_broadcaster.go

```diff
@@ -173,7 +173,13 @@ utxoLoop:
 				return nil, fmt.Errorf("failed to pay transaction %d: %v", amount, err)
 			}
 
-			// Todo: Add OP_RETURN with text "ARC testing" so that WoC can tag it
+			if b.opReturn != "" {
+				err = tx.AddOpReturnOutput([]byte(b.opReturn))
+				if err != nil {
+					return nil, fmt.Errorf("failed to add OP_RETURN output: %v", err)
+				}
+				b.logger.Info("added OP_RETURN output")
```

Comment:
This creates too many logs. Please remove, but add on line 75
```go
b.logger.Info("Start broadcasting", slog.String("wait for status", b.waitForStatus.String()), slog.String("op return", b.opReturn))
```

Reply:
Done